content/articles/2018-02-devtools-refactor.Rmd

```diff
@@ -0,0 +1,65 @@
+
+[usethis](http://usethis.r-lib.org/) is the first of these more focused packages to be released publicly on CRAN. usethis is the home for functions related to package and project setup. Writing the blog post for its recent CRAN release (ADD LINK) inspired us to clarify the devtools situation in this post.
+
+Other packages in development on GitHub, but not yet released on CRAN:
+
+  * FILL
+  * THIS
+  * IN?
+
+## Long live devtools!
+
+devtools has a bright future and is an important package for the r-lib/tidyverse team. devtools will continue to exist as a meta-package, exposing functionality from lower-level, domain-specific packages. The implementation will be different from the [tidyverse meta-package](http://tidyverse.tidyverse.org), but the spirit is the same.
+
+In steady state, if you use functions like `load_all()` or `install_github()` in your daily workflow, you should attach devtools via `library("devtools")` and carry on.
+
+There are a few reasons you might want to know more about our internal refactoring:
+
+  * The Awkward Adolescent Phase. It is impossible to choreograph the release of a suite of packages on CRAN. Therefore, there may be periods when functionality exists in usethis, for example, that is not yet exposed by devtools. During such times, you could access those specific functions directly through usethis. If you attach devtools in `.Rprofile`, for interactive use, this implies you should attach devtools first and then any of its constituent packages, as they roll out onto CRAN. Again, this should only be necessary during the transition period.
+  * If you depend on devtools in production scripts or in a package, you should transition that dependency to the lower level package that hosts the functions you need. More below.
+  * If you find a bug, it is most helpful to report it in the repository of package where that function is defined.
+
+## Don't depend on devtools!
+
+Although devtools is here to stay, we'd prefer to see no new direct reverse dependencies. And once the remaining consituent packages are released, we'd like to see existing dependencies transfer away from devtools when possible. devtools has been a *de facto* meta-package for a while and that will become technically true in the coming months. Like the tidyverse meta-package, devtools is convenient for interactive use, but if you are using specific functions programmatically, i.e. in production scripts or in another package, please do so via the function's primary package home and not through devtools.
```

Comment:
* Strike the "And" in favour of "Once the remaining constituent..."?
* Add comma before and in "meta-package for a while*,* and that..."